gcampus/auth/templates/gcampusauth/login_form.html

```diff
@@ -1,62 +1,142 @@
-                            {% include "gcampuscore/forms/components/label.html" with class="mb-2" %}
-                            {{ field }}
-                            {% if field.errors %}
-                                <small class="text-danger">{{ field.errors }}</small>
-                            {% endif %}
-                        </div>
-                    {% endwith %}
-                    {{ form.next_url }}
-                </div>
-                <div class="col-lg-12">
-                    <button class="btn btn-primary btn-block btn-sm w-100">
-                        {% translate "Login" %}
-                    </button>
-                </div>
+{% extends "gcampuscore/base.html" %}
+{% load static i18n auth_token %}
+{% block body %}
+    <div class="d-flex flex-column flex-md-row min-vh-100 g-0 align-items-md-stretch align-items-center">
+        <div class="col-12 col-md-8 bg-white d-flex flex-column justify-content-center">
+            <div class="container text-center mx-auto py-5">
+                <h1 class="mb-4">
+                    {% blocktranslate trimmed %}
+                        Login to
+                        <span class="gcampus-logo text-dark"><span>Gewässer</span>Campus</span>
+                    {% endblocktranslate %}
+                </h1>
                 {% if authenticated %}
-                    <hr class="my-4">
-                    <small class="text-muted">
-                        {% blocktranslate trimmed %}
-                            If you login with a different access key or course token,
-                            you will automatically be logged out from your old key.
-                        {% endblocktranslate %}
-                    </small>
+                    {% include "gcampusauth/components/current_token_info.html" %}
+                    {% blocktranslate trimmed %}
+                        If you login with a different access key or course token,
+                        you will automatically be logged out from your old key.
+                    {% endblocktranslate %}
+                {% else %}
+                    {% blocktranslate trimmed %}
+                        Login as a student using you <b>access key</b>
+                        to create new measurements and edit your
+                        existing ones.<br>
+                        Teachers can login using a <b>course token</b>
+                        to administer the course, edit measurements and
+                        create new access keys.
+                    {% endblocktranslate %}
                 {% endif %}
             </div>
-        </form>
+            <div class="pb-4 row g-0 justify-content-center">
+                <div class="col-12 col-lg-6 px-lg-3 d-flex justify-content-center align-items-start border-end-lg">
+                    <h2>
+                        {% include "gcampuscore/icons/person-fill.html" with extra_class="text-muted flex-shrink-0 me-3" only %}
```

Comment:
Can you try to move to the icon to the right, make it bigger and lighter grey? (So we could be more consistent (reusing the style of the water quality index cards.)